src/gss_names.c

```diff
@@ -626,9 +708,104 @@ uint32_t gssntlm_inquire_name(uint32_t *minor_status,
                               gss_OID *MN_mech,
                               gss_buffer_set_t *attrs)
 {
-    return GSS_S_UNAVAILABLE;
+    uint32_t retmin;
+    uint32_t retmaj;
+    uint32_t tmpmin;
+    gss_buffer_desc buf;
+    const struct gssntlm_name *in = (const struct gssntlm_name *)name;
+
+    if (name == GSS_C_NO_NAME) {
+        return GSSERRS(GSS_S_BAD_NAME, GSS_S_CALL_INACCESSIBLE_READ);
+    }
+    if (!attrs) {
+        return GSSERRS(ERR_NOARG, GSS_S_CALL_INACCESSIBLE_WRITE);
+    }
+    *attrs = GSS_C_NO_BUFFER_SET;
+
+    if (in->attrs == NULL) {
+        return GSSERRS(0, GSS_S_COMPLETE);
+    }
+
+    for (size_t i = 0; in->attrs[i].attr_name != NULL; i++) {
+        struct gssntlm_name_attribute *attr = &in->attrs[i];
+        size_t attr_name_len = strlen(attr->attr_name);
+
+        gss_buffer_t attr_value = &attr->attr_value;
```

Comment:
move one line up (no empty lines between declarations) and add empty line after.

Reply:
ok